projects/ccf-rui/src/app/modules/content/stage-nav/stage-nav.component.html

```diff
@@ -1,10 +1,19 @@
-<mat-radio-group class="input-group" (change)="updateSide($event.value)">
-  <mat-radio-button value="left" class="nav-input">Left</mat-radio-button>
-  <mat-radio-button value="right" class="nav-input">Right</mat-radio-button>
-  <mat-radio-button value="anterior" class="nav-input" checked>Anterior</mat-radio-button>
-  <mat-radio-button value="posterior" class="nav-input">Posterior</mat-radio-button>
-</mat-radio-group>
+<div class='dropdown-label' (click)="toggleNav()">
+  <mat-label>Options</mat-label>
+  <mat-icon>arrow_drop_down</mat-icon>
+</div>
+<div class='stage-nav' [class.hidden]='stageNavHidden'>
```

Comment:
Slightly strange class name considering this is inside the stage-nav component. A better name might be `options` or something along those lines